pkg/cluster/install.go

```diff
@@ -29,35 +29,75 @@ import (
 
 // AdminUpdate performs an admin update of an ARO cluster
 func (m *manager) AdminUpdate(ctx context.Context) error {
-	steps := []steps.Step{
+	toRun := m.adminUpdate(ctx)
+	return m.runSteps(ctx, toRun)
+}
+
+func (m *manager) adminUpdate(ctx context.Context) []steps.Step {
```

Comment:
ctx is not used at all in this function
```suggestion
func (m *manager) adminUpdate() []steps.Step {
```

Reply:
fixed